tests/tools/oecertdump/host/sgx_quote.cpp

```diff
@@ -443,26 +443,28 @@ oe_result_t generate_sgx_report(oe_enclave_t* enclave, bool verbose)
 
             oe_sgx_endorsement_item tcb_issuer_chain =
                 endorsements.items[OE_SGX_ENDORSEMENT_FIELD_TCB_ISSUER_CHAIN];
-            log("Endorsement: Revocation TCB Issuer Chain:\n");
+            log("Endorsement: Revocation TCB Issuer Chain:\n%s\n\n",
+                tcb_issuer_chain.data);
             output_certificate_chain(
                 tcb_issuer_chain.data, tcb_issuer_chain.size, false);
 
             oe_sgx_endorsement_item crl_pck_cert =
                 endorsements.items[OE_SGX_ENDORSEMENT_FIELD_CRL_PCK_CERT];
-            log("Endorsement: CRL PCK Certificate:\n");
+            log("Endorsement: CRL PCK Certificate:\n%s\n\n", crl_pck_cert.data);
             decode_crl_pem(crl_pck_cert.data, crl_pck_cert.size);
```

Comment:
Is this going to break if the crl is der encoded?

Reply:
No. If input buffer cannot be decoded as pem, it will print nothing. So we can tell whether input buffer is a pem or a der.